app/views/service-user-details/search.html

```diff
@@ -0,0 +1,55 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  MoJ Interventions - make referrals
+{% endblock %}
+
+{% block beforeContent %}
+
+  {% from 'govuk/components/phase-banner/macro.njk' import govukPhaseBanner %}
+  {{ govukPhaseBanner({
+    tag: {
+      text: "prototype"
+    },
+    html: 'This is a new service - your <a href="#" class="govuk-link">feedback</a> will help us to improve it.'
+  }) }}
+
+  {{ govukBackLink({
+    text: "Back",
+    href: "../referral-task-list"
+  }) }}
+
+{% endblock %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <form action="/service-user-details/show" class="form">
+        {{ govukInput({
+          label: {
+            text: "Enter service user CRN",
+            classes: "govuk-label--xl",
+            isPageHeading: true
+          },
+          hint: {
+            text: "Searching for a user requires a running instance of the Community API"
```

Comment:
Our users may not know about the Community API -- what do you think about mentioning nDelius instead?

Reply:
Have added this to the page title.